astropy/table/index.py

```diff
@@ -782,18 +782,18 @@ def __init__(self, table):
         if len(self.indices) == 0:
             raise ValueError("Cannot create TableLoc object with no indices")
 
-    def __getitem__(self, item):
+    def __get_row(self, item):
         '''
-        Retrieve Table rows by value slice.
+                Retrieve Table rows indexes by value slice.
 
-        Parameters
-        ----------
-        item : column element, list, ndarray, slice or tuple
-            Can be a value of the table primary index, a list/ndarray
-            of such values, or a value slice (both endpoints are included).
-            If a tuple is provided, the first element must be
-            an index to use instead of the primary key, and the
-            second element must be as above.
+                Parameters
+                ----------
+                item : column element, list, ndarray, slice or tuple
+                    Can be a value of the table primary index, a list/ndarray
+                    of such values, or a value slice (both endpoints are included).
+                    If a tuple is provided, the first element must be
+                    an index to use instead of the primary key, and the
+                    second element must be as above.
         '''
```

Comment:
Please revert the indentation to a more standard form.

Reply:
Also not sure that `_get_rows` needs a detailed docstring. Having the docstring on `__getitem__` is enough for me.